packages/api/src/utils/server/parseAcceptHeader.ts

```diff
@@ -0,0 +1,75 @@
+import {ResponseFormat} from "../../interfaces.js";
+
+enum MediaType {
+  json = "application/json",
+  ssz = "application/octet-stream",
+}
+
+export const MEDIA_TYPES: {
+  [K in ResponseFormat]: MediaType;
+} = {
+  json: MediaType.json,
+  ssz: MediaType.ssz,
+};
+
+function responseFormatFromMediaType(mediaType: MediaType): ResponseFormat {
+  for (const k of Object.keys(MEDIA_TYPES) as ResponseFormat[]) {
+    if (MEDIA_TYPES[k] == mediaType) {
+      return k;
+    }
+  }
```

Comment:
I like what you did here but the spec only calls for two options so it feels like a switch with default case will be more performant.  What are your thoughts?

Reply:
The goal of the loop was to avoid adding touch points that need to be changed if the media type set changes.

With the loop, a new format added in `ResponseFormat` only need a media type to be added in `enum MediaType` and the `MEDIA_TYPES` mapping. 

If this becomes a switch statement with a default case, and a new format/media type are added, it would be easy to forget to update the switch and accidentally return json.

That said, it doesn't seem very important, given that it is unlikely any formats other than json and ssz will be supported in the foreseeable future, so whatever the maintainers think is easiest is fine with me.

Reply:
In many circumstances I think this would be the correct approach but its not likely that the spec will change for these to include types other than `json` and `ssz`